### ISSUE TITLE
Swagger file empty

### ISSUE THREAD

**Issue by clounie** [Bankdata/gradle-swagger-plugin#2]
I can't get this plugin to generate the yaml correctly.

My build.gradle looks like so:
```groovy
buildscript {
    repositories {
        mavenCentral()
    }
    dependencies {
        classpath (group: 'dk.bankdata.gradle.swagger', name: 'gradle-swagger-plugin', version: '1.0.0'
        )
    }
}
plugins {
    id 'java'
}

apply plugin: 'dk.bankdata.swagger'

swagger {
    resourcePackages = ['net.nw.test']
}

group 'clounie-labs'
version '0.1.0'

sourceCompatibility = 1.8

repositories {
    mavenCentral()
}

dependencies {

    // https://mvnrepository.com/artifact/org.apache.commons/commons-lang3
    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.7'

    compile group: 'io.swagger.core.v3', name: 'swagger-jaxrs2', version: '2.0.2'
    compile group: 'io.swagger.core.v3', name: 'swagger-jaxrs2-servlet-initializer', version: '2.0.2'

    compile group: 'org.glassfish.jersey.core', name: 'jersey-server', version: '2.27'
    compile group: 'org.glassfish.grizzly', name: 'grizzly-http-server', version: '2.4.3'
    compile group: 'org.glassfish.jersey.containers', name: 'jersey-container-grizzly2-http', version: '2.27'
    compile group: 'org.glassfish.jersey.inject', name: 'jersey-hk2', version: '2.27'

    testCompile group: 'junit', name: 'junit', version: '4.12'
}
```

My class in `net.nw.test`:
(File path: `{projectDir}/src/main/java/net/clabs/api/TransactionsApi.java`)
```java
package net.nw.test;

import io.swagger.v3.oas.annotations.Operation;
import io.swagger.v3.oas.annotations.Parameter;
import io.swagger.v3.oas.annotations.responses.ApiResponse;
import net.clabs.models.Transaction;

import javax.ws.rs.*;
import javax.ws.rs.core.MediaType;
import javax.ws.rs.core.Response;
import java.util.List;

@Path("/transactions")
@Consumes(MediaType.APPLICATION_JSON)
@Produces(MediaType.APPLICATION_JSON)
public class TransactionsApi {

  @GET
  @Path("/{transactionId}")
  @Operation(summary = "Retrieves a transaction by id",
    responses = {
      @ApiResponse(responseCode = "200", description = "Successfully found and returned the transaction"),
      @ApiResponse(responseCode = "404", description = "Unable to find the specified transaction")
    }
  )
  public Transaction getTransactionById(
    @PathParam("transactionId") String transactionId
  ) {
    return null; // TODO
  }
}
```

Yet...when I `gradle swagger`, all I get in `build/swagger/swagger.yaml` is:
```
---
swagger: "2.0"
```

Any ideas?

**Comment by langecode** [comment]
I think the api class is missing the `@Api` annotation. This marks the class for inclusion as part of the api documentation with the Swagger framework. See for instance https://github.com/Bankdata/gradle-swagger-plugin/blob/master/src/test/groovy/dk/bankdata/gradle/swagger/example/AccountServiceExposure.java

Can you confirm whether this is the case?

**Comment by clounie** [comment]
Sorry, meant to get to this last night - will respond later today.

**Comment by clounie** [comment]
Ended up just trying it now - and got it working!

The new v3 doesn't have the `@Api` annotation, so what I ended up doing is just adding this dependency to gradle for now:
```gradle
// https://mvnrepository.com/artifact/io.swagger/swagger-annotations
compile group: 'io.swagger', name: 'swagger-annotations', version: '1.5.20'
```

This let me add `@Api` to my file and then it creates the yaml file. Here are the results if you're interested: https://gist.github.com/clounie/9506da45dacb3a70ee28d2264639d76f

Not perfect, but considering you haven't tested with that version, not bad at all! The only thing that particularly caught my eye was the missing responses (it seems to ignore 4xx)

For now I'll downgrade to the old annotations, but thanks for your help!
